engine/src/hopeit/app/api.py

```diff
@@ -96,8 +97,27 @@ def _payload_description(arg: PayloadDef) -> str:
     return str(arg)
 
 
+def _method_summary(module: str, summary: Optional[str] = None) -> str:
+    if summary is not None:
+        return summary
+    doc_str = inspect.getdoc(module)
+    if doc_str is not None:
+        return doc_str.split("\n")[0]
+    return ""
+
+
+def _method_description(module: str, description: Optional[str] = None, summary: Optional[str] = None) -> str:
+    if description is not None:
+        return description
+    doc_str = inspect.getdoc(module)
+    if doc_str is not None and doc_str.count('\n') > 1:
+        return doc_str.split("\n", 2)[2]
```

Comment:
I am still not sure about using a fixed line number. Maybe worth try using all lines after title line and strip non alphabetical characters: ...[1:].sub(r"^\\W+", "") to b3 compatible with most docstrings. I need to try with current implemented apps... but feel free to explore and provide options.

Reply:
This is still pending. I still prefer use everything after first newline stripping non alphanumeric chars.

Reply:
@leosmerling-hopeit: You are right, it is better to get from the second line and clean up if it is empty. I already implemented it this way.